streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -212,31 +212,29 @@ public void shouldThrowIfCommittingOnIllegalState() {
     @Test
     public void shouldAlwaysCheckpointStateIfEnforced() {
         stateManager.flush();
-        EasyMock.expectLastCall().once();
+        verify(stateManager, times(1)).flush();
```

Comment:
Again, you make a call on the mock and verify that you made the call. In EasyMock you need to make the calls on the mock to specify what calls you expect. The call to `replay()` ends the specification phase. When you `verify()`, you verify that the call under test indeed called to methods you specified. 
This is not the case in Mockito. In Mockito the mocks record the calls and you verify what was called. Making calls on the mock directly in the test method does not make sense. The methods of the mocks should be called within the call under test.

Reply:
Done.